### ISSUE TITLE
Write a crawler for each completed case page

### ISSUE THREAD

**Issue by rgarner** [rgarner/cma-tna-crawlers#13]
Going through the current cases/year archive does not seem to be enough. There are more cases on these entry pages:

* [Cartels completed cases](http://webarchive.nationalarchives.gov.uk/20140402142426/http://www.oft.gov.uk/OFTwork/competition-act-and-cartels/criminal-cartels-completed/;jsessionid=6A7FF4882DECE807EEDB7FA066727239)
* [Consumer enforcement completed cases](http://webarchive.nationalarchives.gov.uk/20140402142426/http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/;jsessionid=7D9351A4A3A00F44CC929F7F5D63FE27)
* [Completed market studies](http://webarchive.nationalarchives.gov.uk/20140402142426/http://www.oft.gov.uk/OFTwork/markets-work/completed/;jsessionid=C4B0858979A314666A1C5A29D6B837C0)

**Comment by rgarner** [comment]
Existence report generated from https://github.com/rgarner/cma-tna-crawlers/blob/e5f535a44aadc8d769360c818a69442f253779a3/lib/cma/oft/check_completeness_crawler.rb:

```json
{
  "dont_exist": [
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/air-travel/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/carpet-flooring/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/furniture-carpet/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/hanover/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/hart/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/pegasus/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/quick-house-sales/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/university-terms/",
    "http://www.oft.gov.uk/OFTwork/consumer-enforcement/consumer-enforcement-completed/energy-efficiency/",
    "http://www.oft.gov.uk/OFTwork/markets-work/quick-house-sales/",
    "http://www.oft.gov.uk/OFTwork/markets-work/airports",
    "http://www.oft.gov.uk/OFTwork/markets-work/government-ict/",
    "http://www.oft.gov.uk/OFTwork/markets-work/pensions/"
  ]
}
```

... so cartels are ok, but we need to crawl a few consumer/market cases.

**Comment by rgarner** [comment]
Fixed in 7390e43e022bffc7861f89ba6c57af40d3360dc3
